src/plugins/data_view_management/public/components/field_editor/field_editor.tsx

```diff
@@ -601,7 +601,7 @@ export class FieldEditor extends PureComponent<FieldEdiorProps, FieldEditorState
           isInvalid={isInvalid}
           error={isInvalid ? errorMsg : null}
         >
-          <CodeEditor
+          <CodeEditor // TODO theme provided in src/plugins/data_view_management/public/management_app/mount_management_section.tsx
```

Comment:
Works now. The TODO can be removed:

<img width="500" alt="Screenshot 2023-06-28 at 10 20 18" src="https://github.com/elastic/kibana/assets/1415710/2b04065e-9bb0-432d-9e2c-136b5de9dde9">